client/tests/sdk/test_job_handler.py

```diff
@@ -1028,7 +1028,7 @@ def report_handler(self): ...
             "mock_user_module:MockReport.report_handler",
         } == {r.name for r in results}
         assert all([r.status == "success" for r in results])
-        assert {1, 1, 10, 1} == {len(r.task_results) for r in results}
+        assert set([1, 1, 10, 1]) == {len(r.task_results) for r in results}
```

Comment:
Does it meet expectations？
`set([1,1,10,1]) == {1, 10}`

Reply:
updated